docs/gpu.md

```diff
@@ -82,4 +82,7 @@ echo "/run/nvidia/driver/usr/lib/x86_64-linux-gnu" > /etc/ld.so.conf.d/nvidia.co
 ldconfig
 ```
 
-And then, the `nerdctl run --gpus` can run successfully.
+And then, the `nerdctl run --gpus` can run successfully. eg:
+```
+nerdctl run -it --rm --runtime=/usr/local/nvidia/toolkit/nvidia-container-runtime --gpus all nvidia/cuda:12.3.1-base-ubuntu20.04 nvidia-smi
```

Comment:
`--runtime` should not be needed on the "typical" setup?
(When `nvidia-container-runtime` is present in `$PATH`?)

Reply:
i have try ,this not work with no param runtime

![image](https://github.com/user-attachments/assets/e04bdf5b-ae0c-4f30-93ff-5ee34f1536ef)

Reply:
@ktock Can you confirm? Is this a bug?

Reply:
@zgfh Can you confirm `nvidia-container-cli` is in PATH?